lib/ccf/archive.py

```diff
@@ -297,6 +297,24 @@ def available_diffusion_unproc_names(self, subject_info):
         name_list = self._get_scan_names_from_full_paths(dir_list)
         return name_list
 
+		
+    def running_status_dir_full_path(self, subject_info):
+        """
+        Full path to the unprocessed diffusion data resource directory
+        """
+        path = self.subject_resources_dir_full_path(subject_info)
+        path += os.sep + 'RunningStatus'
+        return path		
+ 
+    # def available_running_status_dir_full_paths(self, subject_info):
+        # """
+        # List of full paths to any resources containing unprocessing diffusion scans
```

Comment:
You'll also want to change the comment to reflect that this method returns a list of the full paths to running status directories.